Adyen/UI/View Controllers/AddressLookup/AddressLookupStyle.swift

```diff
@@ -13,5 +13,11 @@ public struct AddressLookupStyle: ViewStyle {
     public var search: AddressLookupSearchStyle = .init()
     public var form: FormComponentStyle = .init()
     
-    public init() {}
+    public init(
```

Comment:
Should we add comment for this public init?

Reply:
Thanks :)